src/main/views/MattermostView.ts

```diff
@@ -288,24 +285,29 @@ export class MattermostView extends EventEmitter {
         return this.view.webContents;
     }
 
-    handleInputEvents = (_: Event, input: Input) => {
-        // Handler for pressing the Alt key to focus the 3-dot menu
-        if (input.key === 'Alt' && input.type === 'keyUp' && this.altLastPressed) {
-            this.altLastPressed = false;
-            clearTimeout(this.altTimeout);
-            WindowManager.focusThreeDotMenu();
-            return;
+    private altPressStatus = false;
```

Comment:
Can this be declared above with the other variables?
Also, this should be initialized in the constructor, not as a local variable.

Reply:
done